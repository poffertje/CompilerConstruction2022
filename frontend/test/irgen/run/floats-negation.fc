extern int printf(char[] format, ...);

int main() {
    float f = 4.0;
    printf("%.3f\n", -f);                   // -4.000
    printf("%f\n", -1.0);                   // -1.000000
    return 0;
}
