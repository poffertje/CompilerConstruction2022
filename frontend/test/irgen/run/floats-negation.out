-4.000
-1.000000
